crates/lovely-core/src/lib.rs

```diff
@@ -60,14 +60,23 @@ impl Lovely {
                 .strip_suffix(".app")
                 .expect("Parent directory of current executable path was not an .app")
                 .replace(".", "_")
+        } else if env::consts::OS == "linux" {
+            PathBuf::from(&args[0])
+                .file_stem()
+                .expect("Failed to get file_stem component of current executable path.")
+                .to_string_lossy()
+                .replace(".", "_")
         } else {
             cur_exe
                 .file_stem()
                 .expect("Failed to get file_stem component of current executable path.")
                 .to_string_lossy()
                 .replace(".", "_")
         };
-        let mut mod_dir = dirs::config_dir().unwrap().join(game_name).join("Mods");
+        let mut mod_dir = dirs::data_dir().unwrap().join(&game_name).join("Mods");
+        if !mod_dir.is_dir() {
+            mod_dir = dirs::data_dir().unwrap().join(game_name).join("Mods");
```

Comment:
did you mean this?

```suggestion
            mod_dir = dirs::config_dir().unwrap().join(game_name).join("Mods");
```

if so then i think it won't result in the behaviour u intended, it will instead use `~/.local/share/Balatro/Mods` if someone created it before running lovely, but on the very first run it'll fallback to `~/.config/...` then create and use that, making `config_dir` the default and `data_dir` an exception

Reply:
Ah, well, almost. I did mean to put `dirs::config_dir` in line 76, not 78, but this approach won't be enough anyway, since it shouldn't use `args[0]` if lovely is already used on system. But just to be sure, the intended "legacy" behaviour is that the Mods are put in ~/.config/love/Mods, right? If so, this approach won't work anyway.

Reply:
the legacy behaviour is mods are in `~/.config/Balatro/Mods`